module/zfs/vdev_mirror.c

```diff
@@ -606,13 +606,20 @@ vdev_mirror_state_change(vdev_t *vd, int faulted, int degraded)
 		vdev_set_state(vd, B_FALSE, VDEV_STATE_HEALTHY, VDEV_AUX_NONE);
 }
 
+static boolean_t
+vdev_mirror_need_resilver(vdev_t *vd, uint64_t offset, size_t psize)
```

Comment:
Sure, we can set `vd->vdev_op_need_resilver = NULL` for mirrors if you prefer.  At the time it felt clearer to register something for both mirrors and raidz.  I'll remove it.